plugins/node/instrumentation-mongoose/src/utils.ts

```diff
@@ -105,6 +105,8 @@ export function handleCallbackResponse(
   let callbackArgumentIndex = 0;
   if (args.length === 2) {
     callbackArgumentIndex = 1;
+  } else if (args.length === 3){
+    callbackArgumentIndex = 2;
```

Comment:
This is the fix for insertMany/bulkWrite with callback and options.  
It also fixed issues with older versions, which I previously dropped from support.